lang/cs/Org.Apache.REEF.Network.Tests/NetworkService/WritableString.cs

```diff
@@ -0,0 +1,92 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Org.Apache.REEF.Wake.Remote;
+
+namespace Org.Apache.REEF.Network.Tests.NetworkService
```

Comment:
Somehow I like this organization better since this way there is a logical separation between RemoteManager, TransportLayer, NetworkService, Commun. Group etc. Otherwise putting their writables in one separate directory (I assume once you change namespace you will put all of them in a separate directory to follow the ocnvention) might be confusing. What do you think?